CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# 2.0
+
+## Additions
+- Add adaptive app icon
+- Add two new themes
+- Add app intro
+- Add about screen
+- Implement support for HSL videos
+- Add theme support for charts. Need openHAB server > 2.1
+
+## Improvements
```

Comment:
Please add as a new feature "add support for image items".

Reply:
This might be misleading. Image items worked in the past.

Reply:
Only image through URL was implemented, not image items.